CMakeLists.txt

```diff
@@ -402,6 +402,10 @@ option(SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING
   "Enable experimental Swift differentiable programming features"
   FALSE)
 
+option(SWIFT_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING_FUTURE_IMPLEMENTATION
```

Comment:
@edymtt what do you think of sinking this option to `stdlib/CMakeLists.txt` since it doesn't impact the compiler at all?

Reply:
That's a good idea -- I missed that there were some options in that file as well

Reply:
Should be addressed in b55f31bd98e9f00dea8d4e418614ce7925fdc367